acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/ibft/IbftMiningAcceptanceTest.java

```diff
@@ -43,28 +44,52 @@ public void shouldMineOnSingleNode() throws IOException {
 
   @Test
   public void shouldMineOnMultipleNodes() throws IOException {
-    final PantheonNode minerNode1 = pantheon.createIbftNode("miner1");
-    final PantheonNode minerNode2 = pantheon.createIbftNode("miner2");
-    final PantheonNode minerNode3 = pantheon.createIbftNode("miner3");
-    final PantheonNode minerNode4 = pantheon.createIbftNode("miner4");
-    cluster.start(minerNode1, minerNode2, minerNode3, minerNode4);
+    final PantheonNode validator1 = pantheon.createIbftNode("miner1");
+    final PantheonNode validator2 = pantheon.createIbftNode("miner2");
+    final PantheonNode validator3 = pantheon.createIbftNode("miner3");
+    final PantheonNode nonValidatorNode = pantheon.createIbftNode("miner4");
+    cluster.start(validator1, validator2, validator3, nonValidatorNode);
 
     final Account sender = accounts.createAccount("account1");
     final Account receiver = accounts.createAccount("account2");
 
-    minerNode1.execute(transactions.createTransfer(sender, 50));
+    validator1.execute(transactions.createTransfer(sender, 50));
     cluster.verify(sender.balanceEquals(50));
 
-    minerNode2.execute(transactions.createIncrementalTransfers(sender, receiver, 1));
+    validator2.execute(transactions.createIncrementalTransfers(sender, receiver, 1));
     cluster.verify(receiver.balanceEquals(1));
 
-    minerNode3.execute(transactions.createIncrementalTransfers(sender, receiver, 2));
+    validator3.execute(transactions.createIncrementalTransfers(sender, receiver, 2));
     cluster.verify(receiver.balanceEquals(3));
 
-    minerNode4.execute(transactions.createIncrementalTransfers(sender, receiver, 3));
+    nonValidatorNode.execute(transactions.createIncrementalTransfers(sender, receiver, 3));
     cluster.verify(receiver.balanceEquals(6));
   }
 
+  @Test
+  @Ignore("Temporarily disabled until ibft new block events are being sent")
+  public void shouldMineOnMultipleNodesEvenWhenClusterContainsNonValidator() throws IOException {
+    final String[] validators = {"validator1", "validator2", "validator3"};
+    final PantheonNode validator1 = pantheon.createIbftNodeWithValidators("validator1", validators);
```

Comment:
name field could/should reference the validators array? [or even put this into a loop?]

Reply:
rest of the code can't reference the validators array, best I could do is use the validator names in that array to create the validator nodes. I've done that but I think the test is less clear afterwards.

can't use stream as we have a checked exception (IOException) to deal with from the createIbft methods.

```
final String[] validatorNames = {"validator1", "validator2", "validator3"};
    final List<PantheonNode> validators = new ArrayList<>();
    for (String name: validatorNames) {
      validators.add(pantheon.createIbftNodeWithValidators(name, validatorNames));
    }
    final PantheonNode nonValidatorNode =
        pantheon.createIbftNodeWithValidators("non-validator", validatorNames);
    validators.add(nonValidatorNode);
    cluster.start(validators);

    final Account sender = accounts.createAccount("account1");
    final Account receiver = accounts.createAccount("account2");

    validators.get(0).execute(transactions.createTransfer(sender, 50));
    cluster.verify(sender.balanceEquals(50));

    validators.get(1).execute(transactions.createIncrementalTransfers(sender, receiver, 1));
    cluster.verify(receiver.balanceEquals(1));

    nonValidatorNode.execute(transactions.createIncrementalTransfers(sender, receiver, 2));
    cluster.verify(receiver.balanceEquals(3));
```